packages/widget/src/hooks/api/use-filtered-opportunities.ts

```diff
@@ -15,7 +15,11 @@ export const useFilteredOpportunities = () => {
 
           if (!isConnected) return defaultFilter;
 
-          return network === o.token.network && defaultFilter;
+          return (
+            (network === o.token.network ||
+              o.token.network.includes("goerli")) &&
```

Comment:
I mean, if you are filtering by Cosmos, isn't this gonna show the `Ethereum Goerli`? 

I guess a more sensible filter is: 
```
(network === o.token.network || `${network}-goerli` === o.token.network
```

Reply:
ah true, forgot about cosmos. fixed